wiki/Article_Style_Guide/en.md

```diff
@@ -1,192 +1,280 @@
+### Clickable images
 
-There should be one space between sections as well as one space between the equal signs and the section name for ease of editing. If any "main article" links or thumb images are used, place them immediately under the section header, and then a space after those before the section content.
+If when you need an image to be clickable, i.e. click on thumbnail to view a full sized version, you can use this syntax:
 
-### Italics
+```md
+[![Alt](thumnbail/link.jpg)](full/link.png "hover")
+```
 
-1.  Any instance of "*osu!*" should be in italics unless it is included in the name of game modes or other services related to the game. *e.g.* osu! Standard or osu!direct.
-    1.  Any other instance of the name of a videogame should also be in italics. *i.e. Elite Beat Agents*.
+- `Alt` is the alternative text; if image fails to load
+- `thumbnail/link.jpg` is the link of the thumbnail
+- `full/link.jpg` is the link of the thumbnail
+- `"hover"` is the hover text; when the mouse cursor is on top of the mouse
 
-2.  Any emphasis (in talk pages, etc.) should be in italics instead of being in bold or uppercase letters. Use italics when introducing or distinguishing among terms.
-    1.  The overuse of emphasis will reduce its effectiveness.
+## Tables
 
-3.  Italics should be used for any occurence of foreign words including latin expressions and abbreviations like *e.g.* and *a priori*.
+Tables use the GFM syntax.
 
-### Images
+If you cannot create a table because _something_ won't work without HTML (i.e. lists inside tables), then you are overthinking both the table and the content.
+If you need help making a complicated table, try to see if you can improvise a simplier fix for it.
 
-Images added to articles should fit the following guidelines:
+**Never** place images inside tables.
 
--   Images should showcase an attribute of the articles topic.
-    -   Images should not have the sole purpose of showcasing a bug, instead report the bug in the [Help subforum](https://osu.ppy.sh/forum/5).
-    -   Images showcasing usage of specific features for decoration should be avoided.
--   When adding screenshots to an article, make sure the screenshots use the default skin. Screenshots that use custom skins or other custom content are not allowed.
-    -   This does not apply to articles covering skinning.
-    -   In the case of Taiko screenshots (Gameplay only), the Taiko skin can be used instead
--   Articles should only have one image showcasing an individual attribute of the articles content. For example, a slider in the beatmap editor.
--   Image captions should not have periods at the end, unless the phrase is a full sentence.
--   Images should showcase the most up to date version of *osu!* available for the content.
-    -   Images that are outdated are subject to be removed.
+## FAQ
```

Comment:
well, this is a FAQ of MediaWiki and GitHub stuff, not really pertaining to osu! (the game and itself)